src/OrbitGl/App.h

```diff
@@ -306,6 +306,13 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
   orbit_base::Future<ErrorMessageOr<void>> RetrieveModuleAndLoadSymbols(
       const std::string& module_path, const std::string& build_id);
 
+  // This method is pretty similar to `RetrieveModule`, but it also requires debug information to be
+  // present. `RetrieveModule` will be satisfied with a symbol section.
```

Comment:
I don't understand the sentence "`RetrieveModule` will be satisfied with a symbol section."

Reply:
Yes, this was weird. I removed it and adjusted the comment for `RetrieveModule` accordingly. It should be clearer now.